### DISCUSSION TITLE
Ajaxify all links except those to pdf and docx files

### DISCUSSION THREAD

**Discussion by zddors** [jiripudil/Naja#308] [Support] [answered]
Hi, I need to ajaxify all links and forms except class noajax and except all links to pdf a docs files. 
Until I used nette.ajax.js I excluded link to pdf/docx by:
`$("a[href$='.pdf'], a[href$='.docx']").netteAjaxOff();`

But I can't figured out, how to do the same with Naja. Can You Help me, please?

**Comment by jiripudil** [comment, marked as answer]
Hello, there is currently no imperative API to achieve this, but I think you should be able to use the selector configuration to exclude these links:

```js
naja.uiHandler.selector = '.ajax:not([href$=".pdf"]):not([href$=".docx"])';
```

To take effect, this must be set before calling `naja.initialize()`.

**Reply by zddors** [reply]
Thank you, it works fine. In my case:
`naja.uiHandler.selector = ':not(.noajax):not([href$=".pdf"]):not([href$=".docx"])';`
